packages/flutter/lib/src/widgets/basic.dart

```diff
@@ -7760,8 +7760,10 @@ class ColoredBox extends SingleChildRenderObjectWidget {
   }
 }
 
-class _RenderColoredBox extends RenderProxyBoxWithHitTestBehavior {
-  _RenderColoredBox({ required Color color })
+/// A render object that paints a [color] as its background.
```

Comment:
"render object" => "[RenderObject]"

Reply:
"[color]" => "[Color]".  I think the lowercase `color` won't link to anything if I'm not mistaken, since it's not directly a `this` property.